### PR TITLE
Move to pkg:lints

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Review by CaseyHillers — APPROVED** [review]
Thanks for the cleanup! LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## 9.0.2-dev
```
Comment (robrbecker) [inline CHANGELOG.md:1]: If you want to publish after this is merged, let's go right to 9.0.2

File: pubspec.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: github
-version: 9.0.1
+version: 9.0.2-dev
```
Comment (robrbecker) [inline pubspec.yaml:2]: same here, straight to 9.0.2

**Review by robrbecker — CHANGES_REQUESTED** [review]
Looks good. If you don't mind, let's go right to version 9.0.2 without the -dev

File: pubspec.yaml (hunk 2 above)
Comment (kevmoo) [reply pubspec.yaml:2]: Zero effect on the shipping product. No rush.

**Pull request merged**